[FIX] remove hardcoded table name

nvn-odoo opened the pull request:
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:




--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

tivisse commented:
Ok it's prettier, but why is it really needed in a stable release ? 

Does it fix something ?

nvn-odoo commented:
Hi i dont think it solve something. But I guess it is easier when there is a model rename. However. I saw the commit message is not correct. I will fix it tomorrow.I am in holliday today. Thank you for fast review

mreficent commented on addons/hr_timesheet_attendance/report/hr_timesheet_attendance_report.py:
```diff
@@ -17,7 +17,7 @@ class TimesheetAttendance(models.Model):
     @api.model_cr
     def init(self):
-        self._cr.execute("DROP VIEW IF EXISTS hr_timesheet_attendance_report")
+        self._cr.execute("DROP VIEW IF EXISTS %s" % self._table)
```
why not call `drop_view_if_exists(self._table)` from `odoo.tools.sql`?

nvn-odoo replied:
Thank you, good idea, now it is done

nvn-odoo commented:
@tivisse Hi, indeed it doesn't fix stuff but I think it is better to align the way the view is created and dropped.
also fixed the commit message. For me, sounds ok of for merge
Have a great day

tivisse commented:
You can improve this is master is you wish. We should avoid noise on a stable release.

See: https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean

nvn-odoo commented:
thank you for the explanation.
Have a great day

nvn-odoo commented:
@fmdl , as promised, here is the created PR.
regards
